Logger plugin makes library un-usable for some users

From a user report, having the Logger plugin bundled in the kitchen sink is causing errors in invocation.

```
[INFO]    2018-07-12T21:28:47.687Z    8fb92fb0-861a-11e8-8d8d-3b2ad23ab021    Resetting dropped connection: metrics-api.iopipe.com
[ERROR]    2018-07-12T21:29:03.28Z    8fb92fb0-861a-11e8-8d8d-3b2ad23ab021    400 Client Error: Bad Request
Traceback (most recent call last):
File "/var/task/iopipe/contrib/logger/request.py", line 29, in upload_log_data
response.raise_for_status()
File "/var/runtime/botocore/vendored/requests/models.py", line 851, in raise_for_status
raise HTTPError(http_error_msg, response=self)
botocore.vendored.requests.exceptions.HTTPError: 400 Client Error: Bad Request
```

--- a/iopipe/constants.py
+++ b/iopipe/constants.py
@@ -6,4 +6,4 @@ MODULE_LOAD_TIME = time.time() * 1000
 PROCESS_ID = str(uuid.uuid4())
 TIMESTAMP_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
 METRIC_NAME_LIMIT = 128
-VERSION = "1.6.2"
+VERSION = "1.6.3"

--- a/iopipe/iopipe.py
+++ b/iopipe/iopipe.py
@@ -1,7 +1,6 @@
 from .agent import IOpipeCore
 
 from .contrib.eventinfo import EventInfoPlugin
-from .contrib.logger import LoggerPlugin
 from .contrib.profiler import ProfilerPlugin
 from .contrib.trace import TracePlugin
 
@@ -13,7 +12,7 @@ class IOpipe(IOpipeCore):
 
     def __init__(self, *args, **kwargs):
         configured_plugins = kwargs.pop("plugins", None)
-        plugins = [EventInfoPlugin(), LoggerPlugin(), ProfilerPlugin(), TracePlugin()]
+        plugins = [EventInfoPlugin(), ProfilerPlugin(), TracePlugin()]
         if configured_plugins is not None and isinstance(plugins, list):
             plugins = plugins + configured_plugins
         kwargs["plugins"] = plugins

--- a/setup.py
+++ b/setup.py
@@ -16,7 +16,7 @@ if sys.version_info[0] == 2:
 
 setup(
     name="iopipe",
-    version="1.6.2",
+    version="1.6.3",
     description="IOpipe agent for serverless Application Performance Monitoring",
     long_description=README,
     author="IOpipe",

--- a/tests/test_iopipe.py
+++ b/tests/test_iopipe.py
@@ -2,21 +2,19 @@ def test_default_iopipe(default_iopipe):
     """Asserts that the default agent comes preloaded"""
     iopipe = default_iopipe
 
-    assert len(iopipe.plugins) == 4
+    assert len(iopipe.plugins) == 3
     assert iopipe.plugins[0].name == "event-info"
-    assert iopipe.plugins[1].name == "logger"
-    assert iopipe.plugins[2].name == "profiler"
-    assert iopipe.plugins[3].name == "trace"
+    assert iopipe.plugins[1].name == "profiler"
+    assert iopipe.plugins[2].name == "trace"
 
 
 def test_default_iopipe_override(default_iopipe_override):
     """Asserts the default agent can be overridden"""
     iopipe = default_iopipe_override
 
-    assert len(iopipe.plugins) == 4
+    assert len(iopipe.plugins) == 3
     assert iopipe.plugins[0].name == "event-info"
-    assert iopipe.plugins[1].name == "logger"
-    assert iopipe.plugins[2].name == "profiler"
-    assert iopipe.plugins[3].name == "trace"
+    assert iopipe.plugins[1].name == "profiler"
+    assert iopipe.plugins[2].name == "trace"
 
-    assert iopipe.plugins[3].auto_measure is False
+    assert iopipe.plugins[2].auto_measure is False

